LayoutTests/imported/w3c/web-platform-tests/resources/testdriver-vendor.js

```diff
@@ -274,11 +279,69 @@ window.test_driver_internal.action_sequence = async function(sources)
         case "pointer":
             pointerSource = source;
             break;
+        case "key":
+            keySource = source;
+            break;
         default:
             throw new Error(`Unknown source type "${source.type}".`);
         }
     }
 
+    if (keySource && pointerSource)
+        throw new Error("testdriver-vendor.js for WebKit does not yet support mixing key and pointer sources");
```

Comment:
There was an attempt at fixing this code to make it work with different types of actions in: https://bugs.webkit.org/show_bug.cgi?id=243272 where we started work on implementing wheel actions.

Reply:
Yeah, we should eventually fix that but for now, this PR will do.